src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/HopTransactionStep.tsx

```diff
@@ -155,20 +160,54 @@ export const HopTransactionStep = ({
 
   const { isFetching, data: tradeQuoteQueryData } = useGetTradeQuotes()
 
+  const feeAsset = useAppSelector(state =>
+    selectFeeAssetByChainId(state, tradeQuoteStep?.sellAsset.chainId ?? ''),
+  )
```

Comment:
Try it, you'll like it :P

```suggestion
  const feeAsset = useSelectorWithArgs(selectFeeAssetById, orderToCancel?.sellAssetId ?? '')
```

Reply:
hell yeah 2751c96d3a